### PR TITLE
Pbi-200-218 : filter email status by email

### PR DESCRIPTION
**Pull request by bryannaufal**
## Describe your changes
- Adding Page Header for email status page
- Put filter by email input and integrate

## Issue ticket number and link
https://linear.app/adudu/issue/ADU-218/integrate-filter-emailed-vendors
https://linear.app/adudu/issue/ADU-200/frontend-filter-emailed-vendors

image

### CODE REVIEW

**Comment by edutjie** [comment]
There is one minor issue from sonarcloud, "Remove this unused import of 'PaginationArgs'." Please solve it 🙏. The rest is good.

File: src/routes/__tests__/email.test.tsx [hunk 1]
```diff
@@ -10,11 +10,31 @@ describe('<EmailPage />', () => {
...
 
+  it('should render the header section with the logo', () => {
+    render(withWrappers(<EmailPage />))
+
+    const logo = screen.getByAltText('Kompas Gramedia Logo Background')
+    expect(logo).toBeInTheDocument()
+    expect(logo).toHaveAttribute('src', '/kompas-gramedia-logo-bg.svg')
+
+    expect(screen.getByText('Email Status')).toBeInTheDocument()
+    expect(screen.getByText('Lorem Ipsum')).toBeInTheDocument()
```
Comment (FreeJ1nG) [inline src/routes/__tests__/email.test.tsx]: Can you please remove this "Lorem Ipsum" part? it's no longer needed

File: src/routes/email.tsx [hunk 2]
```diff
@@ -35,6 +42,25 @@ export default function EmailPage() {
 
   return (
     <div className="flex min-h-screen w-full flex-col items-center">
+      <PageHeader>
+        <Typography variant="h2" className="text-white">
+          Email Status
+        </Typography>
+        <Typography variant="body1" className="text-white">
+          Lorem Ipsum
+        </Typography>
```
Comment (FreeJ1nG) [inline src/routes/email.tsx]: Here too

**Review by FreeJ1nG — CHANGES_REQUESTED** [review]

**Comment by FreeJ1nG** [comment]
LGTM

**Pull request merged**